xAPI-Communication.md

```diff
@@ -2080,12 +2100,12 @@ Converted to 1.0.0:
 
 <a name="Appendix3B"/>
 
-### <a name="B">Appendix B</a>: Table of All Endpoints
+### <a name="B">Appendix B</a>: Table of All Resources
 
-### <a name="B.s1"></a>Required Resource Endpoints
+### <a name="B.s1"></a>Required Endpoint
```

Comment:
And actually, looking at it - it is Resources as the table has the Endpoint and Function